enclone_core/src/defs.rs

```diff
@@ -642,7 +643,7 @@ pub struct TigData {
     pub umi_count: usize, // number of UMIs supporting contig
     pub read_count: usize, // number of reads supporting contig
     pub chain_type: String, // e.g. IGH
-    pub annv: Vec<(i32, i32, i32, i32, i32)>, // V annotation (one or two entries), for V..J
+    pub annv: Vec<Annotation>, // V annotation (one or two entries), for V..J
```

Comment:
Not for this PR, but at some point we should make this an explicit "one or two" data structure rather than a vec.